final_application/remove_handwriting.py

```diff
@@ -0,0 +1,95 @@
+import torch

```

Comment:
Since this file and the background filling script are almost identical, I would suggest a refactor :wink: